sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/HttpUtils.java

```diff
@@ -45,14 +45,14 @@ public static String urlDecode(String url) {
         }
     }
 
-    public static URI toURI(String uri) {
-        try {
-            return new URI(uri);
-        } catch (Exception e) {
-            log.error("failed to parse {}", uri, e);
-            throw new IllegalArgumentException("failed to parse uri " + uri, e);
-        }
-    }
+//    public static URI toURI(String uri) {
```

Comment:
Do we need to keep this commented method ?

Reply:
no. will remove. thanks for pointing out.

Reply:
addressed in 048bec85ce063b82dc76c07ac0b26a18e6b5e0dd